src/dataflow/src/render/join/mod.rs

```diff
@@ -240,11 +240,7 @@ impl JoinBuildState {
             column_map.insert(column, column_map.len());
         }
         let mut equivalences = equivalences.to_vec();
-        for equivalence in equivalences.iter_mut() {
-            equivalence.sort();
-            equivalence.dedup();
-        }
-        equivalences.sort();
+        expr::canonicalize::sort_dedup_equivalences(&mut equivalences);
```

Comment:
Does it make sense to call `fuse` here? Idk what assumptions are made/satisfied by the method call, but it feels like it would either be "none" or "lots" whereas this is "half-way". I do understand that it accurately replaces the code that previously existed.

Reply:
Yeah, I'll replace sort/fuse with one canonicalize. I was just thinking about it this morning as I looked everything over again.